civihr_hrjobcontract_entities/civihr_hrjobcontract_entities.module

```diff
@@ -1194,6 +1194,4 @@ function civihr_hrjobcontract_entities_views_api() {
         'api' => 3,
         'path' => drupal_get_path('module', 'civihr_hrjobcontract_entities') . '/views',
     );
-}
-
-
+}
```

Comment:
there is should be only one new line at the end of any file (unless there is a reason not to ) , please fix this .